doc/source/v2/user/index.rst

```diff
@@ -0,0 +1,275 @@
+******************
+User Documentation
+******************
+
+.. important::
+
+   DataBroker release 1.0 includes support for old-style "v1" usage and
+   new-style "v2" usage.  This section addresses databroker's new "v2" usage.
+   It is still under development and subject to change in response to user
+   feedback.
+
+   For the stable usage "v1" usage, see :ref:`v1_index`. See
+   :ref:`transition_plan` for more information.
```

Comment:
I understand it's a copy of the same block, but from `v2/index.rst`. I think it's OK to keep it twice for now, but we should de-duplicate it at some point.

Reply:
Unfortunately, a limitation of rst is that you cannot include an rst file in another rst file. You can include a _txt_ file inside an rst file (which we do in just one place that I know of, whatsnew.rst) but I'm not sold that the extra ceremony is worth the de-duplication in this case.

Reply:
Agreed.